### PR TITLE
[1LP][RFR] Fixed docs

### PR DESCRIPTION
**Pull request by psav**
Fixed docs

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit e8004048cf283a7b813694efe8592d0be47ba5a2:

`cfme/utils/appliance/__init__.py`:
- :bangbang: Line 2925:5: **E265** *block comment should start with '# '*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2acb923f596c65c8e98a84b805e8ddcfda344ab7:

`cfme/utils/appliance/__init__.py`:
- :bangbang: Line 2925:5: **E265** *block comment should start with '# '*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e09e3aa5b75f7617e4a4f61b0627c20edd2c1933:

`cfme/utils/appliance/__init__.py`:
- :bangbang: Line 2925:5: **E265** *block comment should start with '# '*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/utils/dockerbot/build-container.py [hunk 1]
```diff
@@ -1,10 +0,0 @@
-import subprocess
```
Comment (jamesooden) [inline cfme/utils/dockerbot/build-container.py]: This isn't documentation changes (-;

File: cfme/utils/events_db.py [hunk 2]
```diff
@@ -90,11 +90,12 @@ def process_id(self, target_type, target_name):
     def query_miq_events(self, target_type=None, target_id=None, event_type=None, since=None,
                          until=None, from_id=None):
         """Checks whether an event occured.
+
         Args:
             target_type: What kind of object is the target of the event (MiqServer, VmOrTemplate)
             target_id: What is the ID of the object (or name, see :py:meth:`process_id`).
             event_type: Type of the event. Ideally one of the :py:meth:`all_event_types` but other
-                kinds of events exist too.
+                        kinds of events exist too.
```
Comment (jamesooden) [inline cfme/utils/events_db.py]: **QUESTION:** Are we sure want to do that?   In my old life I would make the colons aligned and what you did makes sense (but pep doesn't like that), but with the colons the place to align the wrapped text becomes different for each arg.   Maybe this is better to read your way though, even if its different on each arg.   Is there a standard that requires/suggests it be this way?

File: cfme/utils/dockerbot/build-container.py (hunk 1 above)
Comment (psav) [reply cfme/utils/dockerbot/build-container.py]: True

**Pull request merged**